client/src/livingdocs-component-app/app.html

```diff
@@ -80,7 +80,7 @@ <h2 show.bind="selectedItem !== undefined && !selectedItem.conf.active">${'livin
         </div>
         <div class="livingdocs-component-item-controls">
           <div class="item-edit">
-            <a href="#/editor/${tool}/${selectedItem.id}" target="_blank">
+            <a href="#/editor/${selectedItem.conf.tool}/${selectedItem.id}" target="_blank">
```

Comment:
@benib is there a reason why we need the `#` in the link to the editor?

Reply:
@manuelroth most probably this is there for the local dev version with the hash routing system to work. We can probably get rid of this.

Reply:
Ah I see. Lets leave it then